packages/requests/src/lib/requests-result.ts

```diff
@@ -198,13 +203,26 @@ export function trackRequestResult<TData>(
             ? options.preventConcurrentRequest
             : true;
 
+        const cacheResponseData = options?.cacheResponseData;
+
         if (
           !options?.skipCache &&
           (result.isSuccess ||
             (preventConcurrentRequest && result.fetchStatus === 'fetching')) &&
           !stale
         ) {
-          return EMPTY;
+          return getRequestResult(key).pipe(
+            switchMap((it) => {
```

Comment:
Why you named it `it`?

Reply:
result is already declared in the upper scope

Reply:
Yes, but what is `it`? We can name it `currentKeyResult`

Reply:
I named it requestResult